R/mock2.R

```diff
@@ -75,21 +75,12 @@
 #' }
 #' ```
 #'
-#' To mock here, you'll need to modify `another_function()` inside the
+#' To mock this function, you'd need to modify `another_function()` inside the
 #' `anotherpackage` package. You _can_ do this by supplying the `.package`
-#' argument:
-#'
-#' ```R
-#' local_mocked_bindings(
-#'   another_function = function(...) "new_value",
-#'   .package = "anotherpackage"
-#' )
-#' ```
-#'
-#' But it's not a great idea to mock a namespace that you don't own because
-#' it affects all code in that package, not just code in your package. Instead,
-#' it's safer to either import the function into your package, or make a wrapper
-#' that you can mock:
+#' argument to `local_mocked_bindings()` but we don't recommend it because
+#' it will affect all code in that package, not just code in your package.
```

Comment:
```suggestion
#' it will affect all calls to `anotherpackage::another_function()`, not just the calls originating in your package.
```

Consider this a weakly held opinion, but somehow it feels more precise to me.